UI/Selenium/Helper/Helper.cs

```diff
@@ -77,11 +78,11 @@ public static void Click(this IWebElement element)
             try
             {
                 element.Click();
-                Logger.Info(" The following element has been clicked "+ element);
+                Logger.InfoWithDate("The following element has been clicked "+ element);
             }
             catch (Exception e)
             {
-                Logger.Error(" Exception has occurred " + e);
+                Logger.Error(e,"Exception has occurred" );
```

Comment:
I need to see more detail, what element?  (Element name and page name.)

Reply:
Do we need this Helper.cs? The methods in this class are not being called anywhere

Reply:
Then it can be removed.

Thank you.